### PR TITLE
[rust] Execute Rust tests in CI

### PR DESCRIPTION
**Pull request by bonigarcia**
### Description
This PR contains a CI workflow to run the Rust tests (i.e., the tests for Selenium Manager) in GitHub Actions.

### Motivation and Context
Currently, the Rust tests are not being executed on GitHub Actions. It would be convenient to run these tests each time a new commit with updated on the `rust` folder.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/ci-rust.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: 'CI - Rust'
+
+on:
+  pull_request:
+    branches:
+      - trunk
```
Comment (diemol) [inline .github/workflows/ci-rust.yml]: I think we can leave the branch out, as we'd like to run tests when code is pushed in a "work-in-progress" branch.

File: .github/workflows/ci-rust.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    paths:
+      - 'rust/**'
+  push:
+    branches:
+      - trunk
```
Comment (diemol) [inline .github/workflows/ci-rust.yml]: Same comment here.

**Review by diemol — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-rust.yml (hunk 1 above)
Comment (bonigarcia) [reply .github/workflows/ci-rust.yml]: I have just committed a second patch removing these lines.

File: .github/workflows/ci-rust.yml (hunk 2 above)
Comment (bonigarcia) [reply .github/workflows/ci-rust.yml]: Also removed.

**Review by diemol — APPROVED** [review]
👍

**Comment by diemol** [comment]
One more thing, can you add https://github.com/SeleniumHQ/selenium/blob/trunk/.github/workflows/ci-java.yml#L6 so it can be triggered manually?

**Comment by bonigarcia** [comment]
Sure, I have just committed another patch for manual trigger.

**Review by diemol — APPROVED** [review]
Thank you

**Comment by titusfortner** [comment]
attn @p0deje for https://github.com/SeleniumHQ/selenium/tree/ci-merge

**Pull request merged**